mne/io/cnt/cnt.py

```diff
@@ -0,0 +1,230 @@
+"""Conversion tool from Neuroscan CNT to FIF
+"""
+
+# Author: Jaakko Leppakangas <jaeilepp@student.jyu.fi>
+#
+# License: BSD (3-clause)
+from os import path
+import datetime
+import calendar
+
+import numpy as np
+
+from ...utils import warn, verbose
+from ..constants import FIFF
+from ..utils import _mult_cal_one
+from ..meas_info import _empty_info
+from ..base import _BaseRaw
+
+
+def read_raw_cnt(input_fname, read_blocks=True, preload=False, verbose=None):
+    """
+
+    Parameters
+    ----------
+    input_fname : str
+        Path to the data file.
+    read_blocks : bool
+        Whether to read data in blocks. This is for dealing with different
+        kinds of CNT data formats.
```

Comment:
This was copied from the eeglab reader. The cnt file has the data in weird blocks that must be read in chunks that are determined by ``channel_offset`` in the header. This parameter allows reading the data in a regular fashion without using the block size. Not sure if this param is needed.